website/static/js/registrationUtils.js

```diff
@@ -11,7 +11,7 @@ var bootbox = require('bootbox');
 var moment = require('moment');
 var History = require('exports?History!history');
 
-require('js/koHelpers');
+var koHelpers = require('js/koHelpers');
```

Comment:
This var isn't used anywhere. We only require `koHelpers` in order to register our custom binding handlers.